Main.java

```diff
@@ -10,6 +11,19 @@
    */
   public static void main(String[] args) {
     // TODO code application logic here
-    
+    Scanner input = new Scanner(System.in);
+    System.out.println("Please enter in the x coordinate of the first point");
+    int x1 =input.nextInt();
+    System.out.println("Please enter in the y coordinate of the first point");
+    int y1 =input.nextInt();
+    System.out.println("Please enter in the x coordinate of the second point");
+    int x2 =input.nextInt();
+    System.out.println("Please enter in the y coordinate of the second point");
+    int y2 =input.nextInt();
+//(y2 -y1)/(x2 - x1) 
+int slope =(y2-y1)/(x2-x1);
```

Comment:
Don't forget to comment your code! Also, if you look at the example, the output is 2.0. This would mean that you should use doubles instead.